src/v2/models/AppState.js

```diff
@@ -131,11 +131,13 @@ export default Model.extend({
 
   shouldReRenderView(transformedResponse) {
     const previousRawState = this.has('idx') ? this.get('idx').rawIdxState : null;
+    
     const identicalResponse = _.isEqual(
-      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'refresh']),
-      _.nestedOmit(previousRawState, ['expiresAt', 'refresh']));
-    const isSameRefreshInterval = _.isEqual(_.omit(transformedResponse.idx.rawIdxState, 'expiresAt'),
-      _.omit(previousRawState, 'expiresAt') );
+      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'refresh', 'stateHandle'], {}, true),
+      _.nestedOmit(previousRawState, ['expiresAt', 'refresh', 'stateHandle'], {}, true));
+    const isSameRefreshInterval = _.isEqual(
+      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'refresh', 'stateHandle'], {}, true),
```

Comment:
Why to omit `'refresh'` for `isSameRefreshInterval` ??

Reply:
This should be 
```suggestion
      const isSameRefreshInterval = _.isEqual(
      _.nestedOmit(transformedResponse.idx.rawIdxState, ['expiresAt', 'stateHandle'], {}, true),
      _.nestedOmit(previousRawState, ['expiresAt', 'stateHandle'], {}, true));
```
@anatoliiboiarshchykov-okta please remove `refresh`. `isSameRefreshInterval` basically says if we remove the keys listed (that don't include refresh) and the two IDX responses are still the same, then the refresh interval is the same.

Reply:
typo